api/lib/infrastructure/repositories/badge-acquisition-repository.js

```diff
@@ -2,14 +2,22 @@ const Bookshelf = require('../bookshelf');
 const bookshelfToDomainConverter = require('../utils/bookshelf-to-domain-converter');
 const BookshelfBadgeAcquisition = require('../orm-models/BadgeAcquisition');
 const DomainTransaction = require('../DomainTransaction');
+const bluebird = require('bluebird');
 
 module.exports = {
 
-  async create(badgeAcquisitionsToCreate = [], domainTransaction = DomainTransaction.emptyTransaction()) {
+  async createOrUpdated(badgeAcquisitionsToCreate = [], domainTransaction = DomainTransaction.emptyTransaction()) {
```

Comment:
```suggestion
  async createOrUpdate(badgeAcquisitionsToCreate = [], domainTransaction = DomainTransaction.emptyTransaction()) {
```

**Suggestion :** Pour garder une cohérence au niveau des temps.